cluster/manifests/deletions.yaml

```diff
@@ -10,6 +10,13 @@ pre_apply:
   kind: Deployment
 {{ end }}
 
+# step 2
```

Comment:
Will step 2 be a single PR?

Reply:
Yes, adding the deletions here and making the label selector changes in the statefulset will be done in a single PR.